cmd/cli/plugin-admin/codegen/main.go

```diff
@@ -7,8 +7,8 @@ import (
 	"github.com/aunum/log"
 
 	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo"
 	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/plugin"
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo"
```

Comment:
How about leaving ```buildinfo``` as it is in this PR and if needed make changes in subsequent PRs? I thin Harish is  making this change in https://github.com/vmware-tanzu/tanzu-framework/pull/3260 too.